### PR TITLE
update router scene

### PR DESCRIPTION
**Pull request by festinalente91**
CAUTION! Do no merge yet! I just want to share code before merge.
- update react-native-router-flux to use constants in containers/allLayout.js
- delete all setCurrentScene() which is not in the constructor. (except in camera part).

### CODE REVIEW

**Comment by ochanje210** [comment]
@yuda1124 do not approve this please

File: shared/containers/allLayout.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ const styles = StyleSheet.create({
   }
 });
 
+<<<<<<< HEAD
```
Comment (ochanje210) [inline shared/containers/allLayout.js]: omg delete this lol

File: shared/components/card.js [hunk 2]
```diff
@@ -274,7 +273,6 @@ Card.propTypes = {
   })),
   getDetailImage: PropTypes.func,
   currentScene: PropTypes.string,
-  setCurrentScene: PropTypes.func,
```
Comment (ochanje210) [inline shared/components/card.js]: don't forget to run `npm test` before you commit

**Comment by festinalente91** [comment]
I think this PR is too big and the "setCurrentScene" issue is to sensitive, so I decided to make this PR smaller.

**Pull request closed without merging**